### ISSUE TITLE
Fix download bugs in door43 client

### ISSUE THREAD

**Issue by neutrinog** [unfoldingWord-dev/ts-desktop#985]
words referenced in obs are wrong some times (possibly corrupt from api).
things keep missing when running batch downloads.
some times not cleaning up after downloads.
some times .download files are left without finishing.

validate that container content matches what we have on config.yml.

**Comment by neutrinog** [comment]
we might be able to provide a short term solution by indexing everything in series rather that in parallel.

**Comment by neutrinog** [comment]
We have determined this is a windows only issue and seems to be contained to just the pre-indexing of everything. Updating the content in the app seems to be working reliably.

Since building is primarily automated on linux we are not going to fix this until we have reason to do so.

**Comment by neutrinog** [comment]
@cckozie I'm closing this, but wanted to draw your attention to it in-case you wanted to push it through a different work flow.
We won't fix this unless we find a good reason to do so. See comments above.

**Comment by neutrinog** [comment]
@klappy we probably need some workflow defined for when we run into issues that we determine do not need to be fixed. Something that allows the decision to be reviewed rather than the issue disappearing into the ether.
